Projects/hCore/Sources/Date+LocalDateString.swift

```diff
@@ -25,6 +25,10 @@ extension Date {
         return DateFormatters.displayddMMM.string(from: self)
     }
 
+    public var dateYYYYFormat: String? {
```

Comment:
@sladan-hedvig Why are some of these date formatters returning optional and some not? I think we should be consistent if there are not any special reason for this

Reply:
It shouldn't be optional :S lets fix that in another branch
https://media.tenor.com/HEMQmYHww6MAAAAd/borat-borat-very-nice.gif